consensus/helpers.go

```diff
@@ -40,17 +38,17 @@ func (m *consensusModule) getQuorumCertificate(height uint64, step typesCons.Hot
 	var pss []*typesCons.PartialSignature
 	for _, msg := range m.messagePool[step] {
 		if msg.GetPartialSignature() == nil {
-			m.nodeLog(typesCons.WarnMissingPartialSig(msg))
+			m.logger.Warn().Msg(typesCons.WarnMissingPartialSig(msg))
 			continue
 		}
 		if msg.GetHeight() != height || msg.GetStep() != step || msg.GetRound() != round {
-			m.nodeLog(typesCons.WarnUnexpectedMessageInPool(msg, height, step, round))
+			m.logger.Warn().Msg(typesCons.WarnUnexpectedMessageInPool(msg, height, step, round))
```

Comment:
Similarly to message above and this probably applies to all calls from `typesCons` package.. 

`WarnUnexpectedMessageInPool` currently returns a string.
```
func WarnUnexpectedMessageInPool(_ *HotstuffMessage, height uint64, step HotstuffStep, round uint64) string {
	return fmt.Sprintf("[WARN] Message in pool does not match (height, step, round) of QC being generated; %d, %s, %d", height, StepToString[step], round)
}
```

Instead, I would log this as:
```
m.logger.Warn().Str("step", StepToString[msg.GetStep()]).Uint64("height", height).Uint64("round", round).Msg("Message in pool does not match (height/step/round) of QC being generated")
```